.github/copilot-instructions.md

```diff
@@ -0,0 +1,114 @@
+# .NET IoT — Repository Custom Instructions for GitHub Copilot
+
+> Purpose: steer Copilot to propose changes that match this repo’s structure, coding conventions, and contribution norms for **System.Device.Gpio** and **Iot.Device.Bindings**.
+
+---
+
+## Scope & context (what this repo is)
+- This repo contains the low-level **System.Device.Gpio** APIs and a large collection of device drivers under **Iot.Device.Bindings**.
+- Code is used across different OSes and boards. Prefer **hardware-agnostic** code, with platform-specific behavior isolated behind drivers/abstractions.
+- Treat public APIs as **experimental but stable-in-spirit**: prefer additive changes, avoid breaking patterns unless maintainers direct otherwise.
```

Comment:
for System.Device.Gpio binary breaking changes should not be done unless fixed issue has owners of the repo saying that it should be done. For Iot.Device.Bindings breaking changes can be made but if they can be easily avoided they should be avoided (i.e. adding APIs). If there is clear benefit for doing them in Iot.Device.Bindings then it should be done.